MIGRATION.md

```diff
@@ -116,9 +117,26 @@ The Parse ObjC SDK will be phased out in the future in favor of the more modern
 
 # Known Issues
 
-These issues below are especially important to consider before migrating.
+The following issues are important to consider before migrating.
 
-- After migrating an app to the Parse Swift SDK, launching the app will create a new `Installation` object with a new `installationId`. It will appear as if the app had been uninstalled and then reinstalled, even though it was only updated with the new Parse Swift SDK.
+## Installation
+
+After migrating an app to the Parse Swift SDK, launching the app will create a new `_Installation` object with a new `installationId`. It will appear as if the app had been uninstalled and then reinstalled, even though it was only updated with the new Parse Swift SDK.
+
+This may be problematic if the installation object is directly referenced in your app or if it contains fields that should be maintained, like the `badge` number field for example. To address this, here are two options:
```

Comment:
If the app references the current installation object for any purpose, that reference will be incorrect, because that's not the current installation anymore, even though the user didn't reinstall.

Same issue with data like the badge counter, which will suddenly become zero for example, which has an effect on the user as well.